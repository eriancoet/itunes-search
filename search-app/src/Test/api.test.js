import React from 'react';
import renderer from 'react-test-renderer';
import DisplayResults from "../Components/Display";

describe('<DisplayResults/>', () => {
    it('should render correctly in "debug" mode', () => {
      const component = renderer.create( <DisplayResults/>).toJSON();
    
      expect(component).toMatchSnapshot();
    });
  });
  
  