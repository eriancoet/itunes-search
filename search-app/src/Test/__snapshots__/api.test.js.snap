// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DisplayResults/> should render correctly in "debug" mode 1`] = `
<div
  className="result"
>
  <h2>
    Search Results
  </h2>
  <p>
    Search term: 
    <b />
    , Media type:
     
    <b />
  </p>
</div>
`;


